tests/integration-tests/tests/common/utils.py

```diff
@@ -31,7 +31,7 @@
 
 SYSTEM_ANALYZER_SCRIPT = pathlib.Path(__file__).parent / "data/system-analyzer.sh"
 
-RHEL_OWNERS = ["309956199498", "841258680906", "219670896067"]
+RHEL_OWNERS = ["309956199498", "841258680906", "219670896067", "255153437490", "279693163583"]
```

Comment:
Are these accounts in  isolated regions only?

Reply:
yes